sdk.yaml

```diff
@@ -396,6 +396,16 @@ REST:
       Member List:
         .documentation: https://ably.com/docs/rest/presence#member-list
         .specification: RSP3
+    Publish:
+      .specification: RSL1
+      .synopsis: |
+        Send a message on this channel.
+      Idempotence:
+        .specification: [RSL1k, RTL6a1, TO3n]
+        .synopsis: |
+          Using the `idempotentRestPublishing` client option:
+          Overrides the client library default, from version 1.2 of the features specification,
```

Comment:
Is it necessary to qualify from what version of the spec a given feature is available in this context?

Reply:
Good point, thanks. I shall adjust this wording.

Reply:
https://github.com/ably/features/pull/27/commits/e87be4b78511284561ff41964f6a89e7ac27f032